docs/source/guide/webhook_reference.md

```diff
@@ -0,0 +1,198 @@
+
+| Key | Type | Description |
+| --- | --- | --- | 
+| name | string | Name of the action. In this case, `Annotation created`. | 
+| id | integer | ID of the created annotation. | 
+| result | JSON | JSON format of the annotation created. |
+| task | JSON | The task that the annotation was created for. |
+| completed_by | JSON | Email address of the user that created the annotation. |
+| was_cancelled | boolean | Whether or not the annotation is the result of a skipped task, and an empty annotation. |
+| ground_truth | boolean | Enterprise only. Whether or not the annotation is a ground truth. 
+| created_at | datetime | Date and time that the annotation was created. |
+| updated_at | datetime | Date and time that the annotation was last updated. |
+| prediction | JSON dictionary | Details of the prediction viewed at the time of annotation, if one exists. |
+| project | JSON dictionary | All fields related to the associated project. |
+
+### Example payload
+
+```json
+
+
+```
+
+## Annotation Updated
+
+Sent when an annotation is updated. See how to [set up a webhook for this event](webhooks.html)
+
+### Webhook payload details
+
+| Key | Type | Description |
+| --- | --- | --- | 
+| name | string | Name of the action. In this case, `Annotation created`. | 
+| id | integer | ID of the created annotation. | 
+| result | JSON | JSON format of the annotation created. |
+| task | JSON | The task that the annotation was created for. |
+| completed_by | JSON | Email address of the user that created the annotation. |
+| was_cancelled | boolean | Whether or not the annotation is the result of a skipped task, and an empty annotation. |
+| ground_truth | boolean | Enterprise only. Whether or not the annotation is a ground truth. 
+| created_at | datetime | Date and time that the annotation was created. |
+| updated_at | datetime | Date and time that the annotation was last updated. |
+| prediction | JSON dictionary | Details of the prediction viewed at the time of annotation, if one exists. |
+| project | JSON dictionary | All fields related to the associated project. |
+
+### Example payload
+
+```json
+
+
+```
+
+## Annotation Deleted
+Sent when an annotation is deleted. See how to [set up a webhook for this event](webhooks.html)
+
+### Webhook payload details
+
+| Key | Type | Description |
+| --- | --- | --- | 
+| name | string  | Name of the action. In this case, `Annotation deleted` | 
+| id | integer | ID of the deleted annotation. | 
+
+### Example payload
+
+```json
+{
+    "action": "Annotation deleted",
+    "annotations": [
+        {"id": 1}
+    ]
+}
+```
+
+## Project Created
+
+Sent when a project is created. See how to [set up a webhook for this event](webhooks.html)
+
+### Webhook payload details
+
+| Key | Type | Description |
+| --- | ---  | --- |
+| name | string  | The action that triggered the event. In this case, `Project created`. |
+| project | JSON dictionary | All fields related to the project that was created. | 
```

Comment:
is this detailed enough? is the response the same as the response to https://labelstud.io/api#operation/api_projects_create ? if so I will add a link

Reply:
(Added anyway)